continuous_integration/circle_ci_test.sh

```diff
@@ -0,0 +1,3 @@
+#!bin/bash
+source activate testenv
+make clean test test-coverage
```

Comment:
> In continuous_integration/circle_ci_test.sh:
> I think that we can remove this file, as we are no longer using it, right ?
Ah yes, I forgot this one. Will do it when time permits. In the mean time, I'll make conda less verbose using --quiet option. We are getting closer !